Python/PRP/pyCreatableConvert.cpp

```diff
@@ -232,6 +239,7 @@
 #include "PRP/Modifier/pyMaintainersMarkerModifier.h"
 #include "PRP/Modifier/pyModifier.h"
 #include "PRP/Modifier/pyOneShotMod.h"
+#include "PRP/Modifier/pyPostEffectMod.h"
 #include "PRP/Modifier/pyPythonFileMod.h"
 #include "PRP/Modifier/pyResponderModifier.h"
 #include "PRP/Modifier/pySpawnModifier.h"
```

Comment:
I had them, and I think they all got wiped out in the refactor when that whole thing was replaced, which is why only the header includes are here.  We used to be able to simply uncomment them out.  

Is there a preference for placement/order?  Do all plCreatables go in the switch statement?